ddp-workspace/projects/ddp-sdk/src/lib/components/prism/prism.component.ts

```diff
@@ -0,0 +1,129 @@
+import { AfterViewInit, Component, Input, OnChanges, OnDestroy, OnInit, SimpleChanges, ViewChild } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { debounceTime, distinctUntilChanged, switchMap, tap, map, takeUntil } from 'rxjs/operators';
+import { SearchParticipant } from '../../models/searchParticipant';
+import { ParticipantsSearchServiceAgent } from '../../services/serviceAgents/participantsSearchServiceAgent.service';
+import { SessionMementoService } from '../../services/sessionMemento.service';
+import { enrollmentStatusTypeToLabel } from '../../models/enrollmentStatusType';
+import { Router } from '@angular/router';
+import { MatPaginator } from '@angular/material/paginator';
+import { MatSort } from '@angular/material/sort';
+import { MatTableDataSource } from '@angular/material/table';
+import { Subject, of } from 'rxjs';
+
+const BASE_COLUMNS = ['guid', 'shortId', 'userName', 'email', 'enrollmentStatus', 'dashboardLink'];
+const INVITATION_COLUMNS = ['invitationId'];
+const PROXY_COLUMNS = ['proxyGuid', 'proxyShortId', 'proxyUserName'];
+const LEGACY_COLUMNS = ['legacyAltPid', 'legacyShortId'];
+
+@Component({
+  selector: 'ddp-prism',
+  templateUrl: './prism.component.html',
+  styleUrls: ['./prism.component.scss'],
+})
+export class PrismComponent implements OnInit, OnDestroy, AfterViewInit, OnChanges {
+  @Input() dashboardRoute: string;
+  @Input() withProxy = false;
+  @Input() withInvitationId = false;
+  @Input() withLegacyId = false;
```

Comment:
Instead of @Input() params, let's try to drive the columns that are hidden/shown from sdk configuration service. Idea is that we can just set some configuration parameters and not have to write additional code to pick columns.

Reply:
done!